src/librustc/diagnostics.rs

```diff
@@ -168,6 +168,29 @@ variant constructors or struct constructors (for unit or tuple structs). This
 is because Rust currently does not support compile-time function execution.
 "##,
 
+E0018: r##"
+The value of static and const variables must be known at compile time. You
+can't cast an integer as a pointer because we can't know what value will be
+at the address.
```

Comment:
This looks great except for the second sentence here. I think `integer as a pointer` should be `pointer as an integer ` and `what value will be at the address` should be `what value the address will take`.